pandas/tests/types/test_inference.py

```diff
@@ -946,6 +949,12 @@ def test_nan_to_nat_conversions():
         assert (s[8].value == np.datetime64('NaT').astype(np.int64))
 
 
+def test_is_scipy_sparse(spmatrix):  # noqa: F811
```

Comment:
That's the thing with DSLs. Everyone needs to learn them, including linters. :smiley: